### PR TITLE
Upgraded to Jena version 4.0.0. Fixed imports and removed deprecated …

### PR DESCRIPTION
**Pull request by scferrada**
…API calls. All tests run. It should use Java 11 now.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,12 +38,12 @@
 		<groupId>org.apache.jena</groupId>
 		<artifactId>apache-jena-libs</artifactId>
 		<type>pom</type>
-		<version>3.17.0</version>
+		<version>4.0.0</version>
```
Comment (hartig) [inline pom.xml:41]: You can upgrade directly to 4.1.0

**Review by hartig — COMMENTED** [review]
Thanks Sebastian! However, did you really want to upgrade only to version 4.0.0? The latest version of Jena is 4.1.0.

**Comment by scferrada** [comment]
It uses 4.1 now, it builds correctly and all tests pass.

**Comment by scferrada** [comment]
Merged as instructed in issue #55

**Comment by hartig** [comment]
Thanks Sebastian!

/cc @chengsijin0817

**Pull request merged**